### ISSUE TITLE
free-ethereum.io 

### ISSUE THREAD

**Issue by PatriciaGant** [MetaMask/eth-phishing-detect#5323]
unblock the site it is working fine and your stupid extention blocks site and dont let me acces this site i use for YEARS now so no scam....

**Comment by davidwgrossman** [comment]
apologies! This is a duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/5342
